plans/sep8.md

```diff
@@ -166,39 +164,34 @@ switch (approvalResponse.status) {
     showUser(approvalResponse);
     break;
   case ApprovalResponseStatus.action_required:
-    if (isBrowser) {
-      // To avoid popup blockers, the new window has to be opened directly in
-      // response to a user click event, so we need consumers to provide us a
-      // window instance that they created previously. This could also be done in
-      // an iframe or something.
-      const popup = window.open("", "name", "dimensions etc");
-
-      const actionResult = await approvalProvider.fetchActionInBrowser({
-        response: approvalResponse,
-        window: popup,
-      });
+    // Placeholder to show the message regarding why further actions are required.
+    showUser(approvalResponse.message);
 
-      showUser(actionResult);
+    // If the approval service requested specific fields, load the values of those fields without user intervention.
```

Comment:
💡 Without user intervention seems to be a little prescriptive. If you want to give details you could cover more cases. SEP-8 doesn't define what you should do here, so it's really up to the client.
```suggestion
    // If the approval service requested specific fields, load the values of those fields if we know them,
    // or if we'd like to collect them client side before taking the user to the action url.
```

Reply:
https://github.com/stellar/js-stellar-wallets/pull/201/commits/1a6cdfaba143d3b8239db787f945aa5155a3417b